src/vs/platform/contextkey/test/common/parser.test.ts

```diff
@@ -173,7 +172,7 @@ suite('Context Key Parser', () => {
 
 		test(`/foo`, () => {
 			const input = `/foo`;
-			assert.deepStrictEqual(parseToStr(input), "Lexing errors:\n\nUnexpected token '/foo' at offset 0\n\n --- \nParsing errors:\n\nExpected 'true', 'false', '(', KEY, KEY '=~' regex, KEY [ ('==' | '!=' | '<' | '<=' | '>' | '>=' | 'in' | 'not' 'in') value ] but got '/foo' at offset 0.\n");
+			assert.deepStrictEqual(parseToStr(input), "Lexing errors:\n\nUnexpected token '/foo' at offset 0. Did you forget to escape the '/' (slash) character? Put two backslashes before it to escape, e.g., '\\\\/'.\n\n --- \nParsing errors:\n\nUnexpected '/foo' at offset 0.\n");
```

Comment:
This is totally fine but might be "future-fragile". Meaning that you'll encounter a test failure whenever the string/message is tweaked. You could consider a static error code in addition to the messages and use them here

Reply:
Oh, I had to suffer the pain of fragility, but I would like to go about this problem by fixing https://github.com/microsoft/vscode/issues/174251